### ISSUE TITLE
DOC: `pymc.sample` replaced by `pymc.sampling.mcmc.sample` in the documention

### ISSUE THREAD

**Issue by HumphreyYang** [pymc-devs/pymc#7070]
### Issue with current documentation:

[`pymc.sample`](https://www.pymc.io/projects/docs/en/v5.10.0/api/generated/pymc.sample.html#pymc-sample) was replaced by [`pymc.sampling.mcmc.sample`](https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.sampling.mcmc.sample.html#pymc-sampling-mcmc-sample) in the documentation after `v.5.10.0`, but `pymc.sample` can still be called under `pymc-5.10.2`.

**Comment by OriolAbril** [comment]
Everything should be documented at the path we expect users to use the object from. Here it is very clearly pymc.sample.

This was one of the main efforts in improving the docs between v3 and v4, and doing it well allows code samples (in pymc-examples among others) to automatically add links to the docs within the rendered code blocks.

**Comment by HumphreyYang** [comment]
Many thanks for your reply @OriolAbril.

> Everything should be documented at the path we expect users to use the object from. Here it is very clearly pymc.sample.

So in case, if I understand correctly, the documentation should change the [pymc.sampling.mcmc.sample](https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.sampling.mcmc.sample.html#pymc-sampling-mcmc-sample) back to `pymc.sample` like versions prior to 5.10.1, which is the path users should use.

**Comment by OriolAbril** [comment]
Exactly, not only pymc.sample though, most functions in the [sampling page](https://www.pymc.io/projects/docs/en/stable/api/samplers.html) need to be corrected. Are you interested in sending a PR to fix it?
